tests/e2e/tests/regression/workbench/workbench-pipeline.e2e.ts

```diff
@@ -0,0 +1,80 @@
+        await t.hover(settingsPage.commandsInPipelineValue);
+        await t.click(settingsPage.commandsInPipelineInput);
+        await t.typeText(settingsPage.commandsInPipelineInput, pipelineValues[0], { replace: true });
+        // Verify that negative number converted to positive
+        await t.expect(settingsPage.commandsInPipelineInput.value).eql(pipelineValues[1], 'Value is incorrect');
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that only chosen in pipeline number of commands is loading at the same time in Workbench', async t => {
+        await settingsPage.changeCommandsInPipeline(pipelineValues[2]);
+        // Go to Workbench page
+        await t.click(myRedisDatabasePage.workbenchButton);
+        await workbenchPage.sendCommandInWorkbench(commandForSend, 0.01);
+        // Verify that only selected pipeline number of commands are loaded at the same time
+        await t.expect(workbenchPage.loadedCommand.count).eql(Number(pipelineValues[2]), 'The number of sending commands is incorrect');
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that user can see spinner over the disabled and shrunk Run button and grey preloader for each command in Workbench History', async t => {
+        await settingsPage.changeCommandsInPipeline(pipelineValues[2]);
+        // Go to Workbench page
+        await t.click(myRedisDatabasePage.workbenchButton);
+        await workbenchPage.sendCommandInWorkbench(commandForSend, 0.01);
+        // Verify that user can`t start new commands from the Workbench while command(s) is executing
+        await t.expect(workbenchPage.submitCommandButton.withAttribute('disabled').exists).ok('Loading spinner is not displayed for Run button', {timeout: 1000});
+        // Verify that user can see spinner over the disabled and shrunk Run button
+        await t.expect(workbenchPage.runButtonSpinner.exists).ok('Loading spinner is not displayed for Run button', {timeout: 5000});
+        await t.expect(workbenchPage.queryCardContainer.find(workbenchPage.cssDeleteCommandButton).withAttribute('disabled').count).eql(4, 'The number of commands is incorrect');
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that user can interact with the Editor while command(s) in progress', async t => {
+        const valueInEditor = '100';
+        await settingsPage.changeCommandsInPipeline(pipelineValues[2]);
+        // Go to Workbench page
+        await t.click(myRedisDatabasePage.workbenchButton);
+        await workbenchPage.sendCommandInWorkbench(commandForSend);
+        await t.typeText(workbenchPage.queryInput, commandForSend, { replace: true, paste: true });
+        await t.pressKey('enter');
+        // 'Verify that user can interact with the Editor
+        await t.expect(workbenchPage.queryInputScriptArea.textContent).contains(valueInEditor, {timeout: 5000});
+    });
```

Comment:
Can you please add one more test with validation of order results after run commands in pipelines?

Reply:
added 'Verify that command results are added to history in order most recent - on top' test